tests/BUILD

```diff
@@ -40,6 +40,13 @@ rule_test(
   size = "small",
 )
 
+rule_test(
```

Comment:
This rule is redundant. It doesn't test any property that shouldn't already be covered by other tests. It's the `sh_test` below that tests a useful property.

Reply:
Granted.